Mathlib/Combinatorics/SimpleGraph/LapMatrix.lean

```diff
@@ -17,7 +17,7 @@ This module defines the Laplacian matrix of a graph, and proves some of its elem
 * `SimpleGraph.lapMatrix`: The Laplacian matrix of a simple graph, defined as the difference
   between the degree matrix and the adjacency matrix.
 * `isPosSemidef_lapMatrix`: The Laplacian matrix is positive semidefinite.
-* `rank_ker_lapMatrix_eq_card_ConnectedComponent`: The number of connected components in `G` is
+* `card_ConnectedComponent_eq_rank_ker_lapMatrix`: The number of connected components in `G` is
```

Comment:
I cannot add a suggestion via this interface, but note that there's a missing "of" after  "nullspace" in the docstring of `card_ConnectedComponent_eq_rank_ker_lapMatrix`:

> The number of connected components in `G` is the dimension of the nullspace its Laplacian.

You don't have to fix it in this PR, but I figured I'd point it out while I noticed it.

bors d+